src/api/CertificateManagement.ts

```diff
@@ -61,17 +61,19 @@ export default class CertificateManagement extends CommonHyperledger {
             });
     }
 
-    async getOwnEnrollmentId(): Promise<APIResponse<EnrollmentId>> {
+    async getOwnEnrollmentId(): Promise<APIResponse<{ enrollmentId: string; username: string }[]>> {
```

Comment:
We could declare a datatype for the response

Reply:
Implemented in 7c71933